pkg/log/log.go

```diff
@@ -285,6 +355,12 @@ func registeredBackendNames() string {
 	return names
 }
 
+// ListBackends lists the registered logger backends.
+func ListBackends(w io.Writer) {
+	fmt.Fprintf(w, "available logger backends: %s\n",
+		registeredBackendNames())
+}
```

Comment:
Uh, missed this one in the previous review. Is there (planned) use for this function?

Reply:
Yes. It could be used form main() to list available loggers if, for instance, an appropriate command line option is specified. We can't register an option for it and do the listing here, as it must be done after main has entered, to make sure all init()-time registrations are done.

We just don't bother doing any of this now as we only have a single fmt logger.

Reply:
Really a minor thing, but I'd prolly leave this dead (unused) code out for now, until it's used. wdyt?

Reply:
Sure, I can remove it.